content/_help/verify-your-identity/how-to-add-images-of-your-state-issued-id._fr.md

```diff
@@ -1,14 +1,115 @@
+  </h2>
+</div>
+
+Si vous utilisez un ordinateur pour vérifier votre identité, vous pourrez passer à un téléphone pour prendre des photos de votre pièce d'identité.
+
+### Conseils pour prendre des photos claires avec un téléphone
+
+<div class="grid-row grid-gap">
+  <div class="tablet:grid-col">
+    <img alt="La face de la pièce d'identité est centrée dans le cadre de la caméra et placée sur un fond sombre." src="{{ site.baseurl }}/assets/img/help/id-dos-and-donts/id-do-front.png" />
+  </div>
+  <div class="tablet:grid-col">
+    <img alt="Le dos de pièce d'identité est centré dans le cadre de la caméra et placé sur un fond sombre." src="{{ site.baseurl }}/assets/img/help/id-dos-and-donts/id-do-back.png" />
+  </div>
+</div>
+
+{%
+  include components/icon-list.html
+  items=page.do_list
+  size='md'
+  icon_color='green'
+  icon_shape='check_circle'
+%}
+
+### Problèmes courants liés à la prise de photos d'une pièce d'identité
+
+<div class="grid-row grid-gap margin-bottom-6">
+  <div class="tablet:grid-col">
+    {%
+      include components/image-example-incorrect.html
+      image-alt-text="La pièce d'identité est placée sur un fond blanc."
+      image='/assets/img/help/id-dos-and-donts/id-dont-white-background.png'
+      bordered_image=true
+      do_dont_text="À NE PAS FAIRE"
+      text='utiliser un fond blanc'
+    %}
+  </div>
+  <div class="tablet:grid-col">
+    {%
+      include components/image-example-incorrect.html
+      image-alt-text="La pièce d'identité est placée sur un fond texturé."
+      image='/assets/img/help/id-dos-and-donts/id-dont-textured-background.png'
+      bordered_image=true
+      do_dont_text="À NE PAS FAIRE"
+      text='utiliser des arrière-plans texturés à motifs'
+    %}
+  </div>
+</div>
+<div class="grid-row grid-gap">
+  <div class="tablet:grid-col">
+    {%
+      include components/image-example-incorrect.html
+      image-alt-text="Les bords de la pièce d'identité sont tronqués."
+      image='/assets/img/help/id-dos-and-donts/id-dont-cropped.png'
+      bordered_image=true
+      do_dont_text="À NE PAS FAIRE"
+      text="recadrer les bords de votre pièce d'identité"
+    %}
+  </div>
+  <div class="tablet:grid-col">
+    {%
+      include components/image-example-incorrect.html
+      image-alt-text="Il y a des reflets et des ombres projetées sur la pièce d'identité."
+      image='/assets/img/help/id-dos-and-donts/id-dont-shadow.png'
+      do_dont_text="À ÉVITER"
+      text='les reflets et les ombres'
+    %}
+  </div>
+</div>
+
+<div class="page-content__horizontal-rule"></div>
+## Si vous n'avez pas accès à un téléphone équipé d'une caméra, vous pouvez téléverser un fichier ou utiliser un scanner
+* Utilisez un appareil photo haute résolution (8 mégapixels ou plus).  La plupart des appareils photo des téléphones ou des tablettes fonctionneront, mais pas forcément la webcam de votre ordinateur. [Nous vous recommandons d'utiliser votre appareil mobile](#phone-tips).
```

Comment:
The copy in this bullet should be replaced with:

> Suivez les mêmes conseils pour prendre des photos avec un téléphone

And the hyperlinked text should be "conseils pour prendre des photos de votre pièce d’identité avec un appareil photo"

Reply:
I'm pretty sure I got this right ([commit](https://github.com/18F/identity-site/pull/1151/commits/6207e3220963f8cf1c6f85da6fbcd1a48cd719b7)).

Ended up like this:

> Suivez les mêmes [conseils pour prendre des photos avec un téléphone](url)

Reply:
Same as above, I'm not 💯 I got this right ([commit](https://github.com/18F/identity-site/pull/1151/commits/60f1736fb1d2a533542c2ae7da5f0a4b9f3f2741)). Ended up with:

> Suivez les mêmes [conseils pour prendre des photos avec un téléphone
](url)